session/tidb.go

```diff
@@ -98,6 +98,15 @@ func (dm *domainMap) Delete(store kv.Storage) {
 	dm.mu.Unlock()
 }
 
+// ResetForWithTiKVTest is only used in the test code.
+// TODO: Remove domap and storeBootstrapped. Use store.SetOption() to do it.
+func ResetForWithTiKVTest() {
```

Comment:
It's better to place this function into `_test.go` and make it invisible for normal code.

Reply:
Done